pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -1960,67 +1994,6 @@ func (cc *XController) manageQueueJob(ctx context.Context, qj *arbv1.AppWrapper,
 			klog.Infof("[manageQueueJob] Getting completion status for app wrapper '%s/%s' Version=%s Status.CanRun=%t Status.State=%s, pod counts [Pending: %d, Running: %d, Succeded: %d, Failed %d]", qj.Namespace, qj.Name, qj.ResourceVersion,
 				qj.Status.CanRun, qj.Status.State, qj.Status.Pending, qj.Status.Running, qj.Status.Succeeded, qj.Status.Failed)
 
-			// set appwrapper status to Complete or RunningHoldCompletion
-			derivedAwStatus := cc.getAppWrapperCompletionStatus(qj)
```

Comment:
I think this change is good as this good was adding a lot of strain unto API server.

Reply:
I'm not saying this isn't the case but now this function is called in line 1479, so it may still be straining the API server.

Reply:
We need more data to prove this, For now, scale test data from Red Hat show that MCAD is not straining the API server for 1K AWs. Let me define straining - the amount of queries sent to the API server